src/JustEat.StatsD/StatsDUdpTransport.cs

```diff
@@ -57,7 +57,7 @@ public bool Send(IEnumerable<string> metrics)
 
             try
             {
-                data.RemoteEndPoint = GetIPEndPoint();
+                data.RemoteEndPoint = GetIpEndPoint();
```

Comment:
Naming guidelines is usually to have two capitals if the acronym is only two letters in the framework generally, so how it was was correct.